src/util/child.js

```diff
@@ -61,7 +61,7 @@ type ProcessFn = (
 export function spawn(
   program: string,
   args: Array<string>,
-  opts?: child_process$spawnOpts & {process?: ProcessFn} = {},
+  opts?: child_process$spawnOpts & {detached?: boolean, process?: ProcessFn} = {},
```

Comment:
Why do we need to declared `detached`? Is it missing in the flow definition for `child_process$spawnOpts`? In that case, maybe we should make a PR to add it

Reply:
Yes to both :)